packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts

```diff
@@ -205,20 +205,26 @@ export async function __getBulkThreads(
         GROUP BY TT.id
     ), contest_data AS (
       -- get the contest data associated with the thread
-          SELECT
-              TT.id as thread_id,
-              json_agg(json_strip_nulls(json_build_object(
-              'contest_id', CON.contest_id,
-              'contest_address', CON.contest_address,
-              'thread_id', TT.id,
-              'content_id', CA.content_id,
-              'start_time', CON.start_time,
-              'end_time', CON.end_time
-          ))) as "associatedContests"
-          FROM "Contests" CON
-          JOIN "ContestActions" CA ON CON.contest_id = CA.contest_id AND CON.contest_address = CA.contest_address
-          JOIN top_threads TT ON TT.id = CA.thread_id
-          GROUP BY TT.id
+        SELECT
+            TT.id as thread_id,
+            json_agg(json_strip_nulls(json_build_object(
+            'contest_id', CON.contest_id,
+            'contest_name', CM.name,
+            'contest_cancelled', CM.cancelled,
+            'contest_interval', CM.interval,
+            'contest_address', CON.contest_address,
+            'score', CON.score,
+            'thread_id', TT.id,
+            'content_id', CA.content_id,
+            'start_time', CON.start_time,
+            'end_time', CON.end_time
+        ))) as "associatedContests"
+        FROM "Contests" CON
+        JOIN "ContestManagers" CM ON CM.contest_address = CON.contest_address
+        JOIN "ContestActions" CA ON CON.contest_id = CA.contest_id 
```

Comment:
According to the [query plan](https://explain.dalibo.com/plan/d587ab8e5f1e8ffb#plan/node/33), we are doing a sequential scan on `ContestActions` which for now takes next to no time because there are so few contest actions but it will become very expensive in the future. We should index appropriately to avoid this sequential scan. If we want to use the existing primary key then we _must_ filter by `contract_address` first then by `contest_id`.

Reply:
Same thing is true for the other query in this PR.

Reply:
Also to add - this is one of our worst-performing routes. This change will make it even worse for contest communities. We should be trimming this query down not building even more into it.

Reply:
@timolegros @kurtisassad I am merging this PR as it is needed for contest testing. Created ticket that should cover the additional optimisations or refactoring of those queries https://github.com/hicommonwealth/commonwealth/issues/8292